app/src/utils/MadLibs.ts

```diff
@@ -19,65 +22,69 @@ export interface MadLib {
 function getMadLibPhraseText(madLib: MadLib): string {
   let madLibText = "";
   madLib.phrase.forEach((phraseSegment, index) => {
-    if (phraseSegment.constructor === Object) {
-      let selection = madLib.activeSelections[index]
+    if (typeof phraseSegment === "string") {
+      madLibText += phraseSegment;
+    } else {
+      const phraseSelector = phraseSegment as PhraseSelector;
+      let selectionKey: string = madLib.activeSelections[index]
         ? madLib.activeSelections[index]
         : madLib.defaultSelections[index];
-      madLibText += " " + phraseSegment[selection] + " ";
-    } else {
-      madLibText += phraseSegment;
+      madLibText += " " + phraseSelector[selectionKey] + " ";
     }
   });
   return madLibText;
 }
 
-const VARIABLES: Record<string, VariableId> = {
-  0: "covid_cases",
-  1: "covid_deaths",
-  2: "covid_hosp",
-  3: "covid_cases_pct_of_geo",
-  4: "covid_deaths_pct_of_geo",
-  5: "covid_hosp_pct_of_geo",
-  6: "covid_deaths_per_100k",
-  7: "covid_cases_per_100k",
-  8: "covid_hosp_per_100k",
+// @ts-ignore
```

Comment:
Remove this

Reply:
This is actually required- these vars are complaining when I don't include every single VariableId which I don't think we want to do. I'll add a TODO though to investigate